custom_components/tesla_custom/button.py

```diff
@@ -0,0 +1,52 @@
+"""Support for Tesla charger buttons."""
+from custom_components.tesla_custom.const import ICONS
+import logging
+
+from homeassistant.components.button import ButtonEntity
```

Comment:
Let's put a try/catch for an Import error on this since this is a new thing. Please test with a pre 2012.x.x version of HA to ensure we fail gracefully in that case.